artifactory/commands/generic/upload.go

```diff
@@ -177,18 +180,18 @@ func (uc *UploadCommand) upload() (err error) {
 	uc.result.SetSuccessCount(successCount)
 	uc.result.SetFailCount(failCount)
 	if errorOccurred {
-		err = errors.New("upload finished with errors, Please review the logs")
-		return err
+		err = errors.New("upload finished with errors. Review the logs for more information")
+		return
 	}
 	if failCount > 0 {
-		return err
+		return
```

Comment:
This function includes additional return statements that can be turned from `return err` to `return`.

Reply:
In some places, return err must remain otherwise an error message will appear: "result parameter err not in scope return".